target_postgres/db_sync.py

```diff
@@ -593,7 +593,8 @@ def sync_table(self):
         if insertion_method and insertion_method == "truncate":
             if len(insertion_method_tables) == 0: #if no insertion_method_tables specified, remove all tables and repopulate
                 self.query(self.drop_table())
-            elif table_name[1:-1] in insertion_method_tables: #and if table name in insertion_method_tables
+            elif stream in insertion_method_tables: #and if table name in insertion_method_tables, truncate
```

Comment:
Does this mean it will always expect stream name instead of table name? Can we support both?

Reply:
Yes! I figured it would be clearer in the config to have the stream names instead of the table names, since the table names for postgres are different (underscores, lowercase, etc). We could support both if needed.